### ISSUE TITLE
[Python] Implement new Concept Exercise: slicing

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#1393]
This issue describes how to implement the `slicing` concept exercise for the Python track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student what is the slicing operation in Python and how to use it.

## Learning objectives

* perform a slice operation on an iterable object (string, list, set, tuple)
* create a new object slicing another one

## Out of scope

* `slice()` function

## Concepts

* slicing

## Prerequisites

* indexing and negative indexing 
* `string`
* `list`
* `set`
* `tuple`
* iterable objects

## Resources to refer to

- https://railsware.com/blog/python-for-machine-learning-indexing-and-slicing-for-lists-tuples-strings-and-other-sequential-types/

### Hints

- TBD?

### After

- the `slice()` function

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named slicing_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
